requirements.txt

```diff
@@ -1,6 +1,6 @@
 PyYAML>=5.3
-dask==2021.4.1
-distributed==2021.4.1
+dask<=2021.07.1
+distributed<=2021.07.1
```

Comment:
@jperez999 - I am hoping these changes resolve the "dask-version" issue you reported earlier (which was really a "pandas-version" issue as far as I can tell).